jenkins/databricks/params.py

```diff
@@ -68,11 +73,15 @@
     elif opt in ('-i', '--sparkinstallver'):
         base_spark_version_to_install_databricks_jars = arg
 
+if args:
+    script_args = args
```

Comment:
The conditional should not be necessary.  If there are no arguments then `args` is set to an empty list.  We can use `args` directly, and `script_args` is unnecessary.

Reply:
OK, will remove it